README.md

```diff
@@ -35,7 +35,7 @@ You will need two Okta applications, one Web app and one SPA app. Save the clien
 
 The Web app needs a couple of settings in the Developer console:
 Add a `Login redirect URI`: `http://localhost:8080/authorization-code/callback`
-Add a `Logout redirect URI`: `http://localhost:8080/logout/callback`
+Add a `Logout redirect URI`: `http://localhost:8080`
```

Comment:
Does this need to be `http://localhost:8080` or `http://localhost:8080/` ?  I suspect the latter, but am not 100% sure.

Reply:
It actually does not matter (from the backend point of view) but we have it without the `/` as the default in the dev-console. As a developer you can add or leave the trailing `/` with no negative effect.

Reply:
Huh.  I thought that was something the backend would care about since forcing the '/' is something clients do.  TIL.